frappe/utils/data.py

```diff
@@ -706,11 +706,11 @@ def parse_val(v):
 		v = int(v)
 	return v
 
-def fmt_money(amount, precision=None, currency=None):
+def fmt_money(amount, format=None, precision=None, currency=None):
```

Comment:
New parameters should be added at the end, so that non keyword invocations don't break
```suggestion
def fmt_money(amount, precision=None, currency=None, format=None):
```

Reply:
@hasnain2808 changes done :) Please relook when possible.

If this is accepted will likely try and extend it to `get_formatted` if possible.